scripts/qa_crowbarsetup.sh

```diff
@@ -1723,16 +1723,20 @@ function custom_configuration()
                 fi
             fi
 
-            # assign neutron-l3 role to one of SLE12 nodes
+            # assign neutron-network role to one of SLE12 nodes
             if [ -n "$want_sles12" ] && [ -z "$hacloud"] && [ -n "$want_neutronsles12" ] && iscloudver 5plus ; then
-                # 2015-03-03 off-by-default because Failed to validate proposal: Role neutron-l3 can't be used for suse 12.0, windows /.*/ platform(s).
+                # 2015-03-03 off-by-default because Failed to validate proposal: Role neutron-network can't be used for suse 12.0, windows /.*/ platform(s).
                 local sle12node=$(knife search node "target_platform:suse-12.0" -a name | grep ^name: | cut -d : -f 2 | tail -n 1 | sed 's/\s//g')
-                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-l3']" "['$sle12node']"
+                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['$sle12node']"
             fi
 
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-server']" "['cluster:$clusternamenetwork']"
-                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-l3']" "['cluster:$clusternamenetwork']"
+                if iscloudver 5plus ; then
+                    proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['cluster:$clusternamenetwork']"
+                else
+                    proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-l3']" "['cluster:$clusternamenetwork']"
```

Comment:
could it break on GM5 without updates? then use proposal_set_value ...-network || proposal_set_value -l3